src/data/common_textures.inc.c

```diff
@@ -34,8 +34,8 @@ s8 gTLUTFinishLineBanner[] = { // tlut
     #include "assets/finish_line_banner/gTLUTFinishLineBanner.inc.c"
 };
 
-s8 D_0D000200[] = { // tlut
-    #include "textures/common/132B50_00200.rgba16.inc.c"
+s8 gTextureParticleFire[] = { // tlut
```

Comment:
common_texture_fire_particle

Please use this format for the common_textures.inc.c: snake_case, prefix common_

Reply:
oh ok I forget

Reply:
so it's should be the same with gTLUTFinishLineBanner no ?

Reply:
yes

Reply:
same for name of file ?

Reply:
Just for variables in common_textures.inc.c should start with common_

The idea is that when you see a variable that starts with common_ you know that the variable contains .data. And is common for every course, so it's safe to use that variable no matter the context.

Versus if you see `d_luigi_raceway` you know that in-order to safely use this variable you need to do the following::
```c
switch(gCurrentCourse) {
    case LUIGI_RACEWAY:
}
```
Before it's safe to use that variable.